src/iosrv/init.c

```diff
@@ -238,6 +241,17 @@ dss_tgt_nr_get(int ncores, int nr)
 	return nr_default;
 }
 
+static bool
+enable_target_oversubscribe(void)
+{
+	bool oversub = false;
+
+	if (getenv("ENABLE_TARGET_OVERSUBSCRIBE") == NULL)
+		return oversub;
```

Comment:
the above two lines is not needed as d_get_env_bool() can handle it.
and, tgt_oversub_enabled() maybe more align with naming rule.

Reply:
good point - the whole function can be removed, use d_get_env_bool() directly.